test/syscalls/linux/proc_net.cc

```diff
@@ -54,6 +62,188 @@ TEST(ProcSysNetIpv4Sack, CanReadAndWrite) {
   EXPECT_EQ(buf, to_write);
 }
 
+bool get_snmp_metric(std::string snmp, const char *type, const char *item,
+		     uint64_t *p_val) {
+  int idx = -1;
+
+  for (auto const& line : absl::StrSplit(snmp, '\n')) {
+    if (!absl::StartsWith(line, type))
+	    continue;
+
+    std::vector<std::string> fields =
+        absl::StrSplit(line, ' ', absl::SkipWhitespace());
+    EXPECT_TRUE(!fields.empty());
+
+    if (idx == -1) {
+      for (size_t i = 1; i < fields.size(); i++) {
+        if (fields[i].compare(item) == 0) {
+          idx = i;
+          break;
+        }
+      }
+      continue;
+    }
+
+    EXPECT_TRUE(absl::SimpleAtoi(fields[idx], p_val));
+    return true;
+  }
+
+  return false;
+}
+
+TEST(ProcNetSnmp, TcpReset) {
+  uint64_t oldAttemptFails;
+  uint64_t oldActiveOpens;
+  uint64_t oldOutRsts;
+  auto snmp = ASSERT_NO_ERRNO_AND_VALUE(GetContents("/proc/net/snmp"));
+  EXPECT_EQ(get_snmp_metric(snmp, "Tcp", "ActiveOpens", &oldActiveOpens), true);
+  EXPECT_EQ(get_snmp_metric(snmp, "Tcp", "OutRsts", &oldOutRsts), true);
+  EXPECT_EQ(get_snmp_metric(snmp, "Tcp", "AttemptFails", &oldAttemptFails), true);
```

Comment:
igudger: EXPECT_TRUE? Here and below.

https://gvisor-review.git.corp.google.com/c/gvisor/+/18283/6/test/syscalls/linux/proc_net.cc#101

Reply:
Changed to the way igudger suggested above.